### PR TITLE
add VTK types to function spaces

### PR DESCRIPTION
**Pull request by KristofferC**
Adds VTK types and how many dofs per body / polygon / edge / node each functionspace has

### CODE REVIEW

**Comment by KristofferC** [comment]
plz review

File: src/utilities/geometry_types.jl [hunk 1]
```diff
@@ -4,3 +4,29 @@ immutable Dim{T} end
...
 immutable Triangle <: Shape end
 immutable Square <: Shape end
+
+
+immutable MeshNode{dim, T}
+    n::Vec{dim, T}
+end
+
+get_coordinates(n::MeshNode) = n.n
+get_id(n::MeshNode) = n.id
```
Comment (KristofferC) [inline src/utilities/geometry_types.jl:14]: this should go bye

**Comment by KristofferC** [comment]
almost too quick ;)

**Comment by fredrikekre** [comment]
damnit, I trust in your programming skills!

**Pull request merged**